### ISSUE TITLE
pixelation on live IPTV streams, 4k TV

### ISSUE THREAD

**Issue by jamesr130488** [koying/SPMC#868]
Hi

Whenever watching live IPTV when the camera angle changes or pans the picture pixels/blocks brifely before being okay again. I.E following a golf ball on screen the blue sky background is blocky/pixelated, when the ball lands on the green the image pixelates before sorting itself

Im using an Nvidia shield, hard wired internet and sony bravia 4k TV in game mode with all motion items turned off

I followed the recommended shield/spmc video settings from Koying's github page but it has not helped

Anyone else fixed this?

**Comment by JFG90** [comment]
That's nothing to do with spmc it's the stream itself, i notice it when watching sky sports during a football match on my iptv service

**Comment by jamesr130488** [comment]
Doesnt make sense to me though? the stream data shouldnt care/change based on what your viewing, i.e as soon as the camera pans the stream data pixelates?

To me it points more towards a rendering issue of either SPMC or my Shield struggling to render the image required

Rather simplistic view granted, but made sense it my head when i was trying to work out why it was happening

**Comment by JFG90** [comment]
Trust me it's the stream itself, sky sports I have watched a football game on other streams for example BT Sports and its absolutely fine,  I have also tested on my Samsung Smart IPTV app which is designed to play IPTV streams and it does the exact same which suggests its not your shield and its not SPMC

**Comment by jamesr130488** [comment]
fair enough, almost bought a minix u9 hoping it would help but thought id ask around all the forums first

thanks for the input :)
